us_web_design_standards.info

```diff
@@ -1 +1,66 @@
-name = U.S. Web Design Standards
+name        = US Web Design Standards
```

Comment:
Let's change 'US' to 'U.S.' for consistency with https://playbook.cio.gov/designstandards/ et al.